src/main/java/baseball/Validator.java

```diff
@@ -0,0 +1,31 @@
+package baseball;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Validator {
+    public static boolean validPlayerNumbers(String playerNumbers) {
+        if (isValidLengthAndRangeNumber(playerNumbers) && isNotDuplicate(playerNumbers))
+            return true;
+        throw new IllegalArgumentException();
+    }
```

Comment:
`boolean` 으로 반환하면, 메소드를 호출하는 사람은 `true` 와 `false` 를 기대할 것 같아요.
그러나, `false` 값은 반환이 안되니 혼란을 줄 것 같아요.
예외를 발생하는 게 주요 목적인 메소드라면, `boolean` 반환을 하지 않는 건 어떨까요?

```suggestion
    public static void validPlayerNumbers(String playerNumbers) {
        if (!isValidLengthAndRangeNumber(playerNumbers) {
            throw new IllegalArgumentException();
        }
        
        if (!isNotDuplicate(playerNumbers)) {
            throw new IllegalArgumentException();
        }
    }
```

Reply:
> 값 변환 대신 Exception을 던지면 어떨까요

또한 @minnim1010 님께서 이야기 하신 게, 저도 정확히 잘 이해했는지는 모르겠는데, 이런 이야기가 아니었을까 싶어요.

`boolean` 을 반환한다는 건, 유효하지 않을 때의 책임을  호출자한테 넘긴다는 뜻인데요. ( "난 `false` 만 넘겨줄거야" )
다만, 유효하지 않을 때의 책임을 여전히 `boolean` 반환형의 메소드들이 갖고 있으니 어색하다는 뜻으로 받아들였어요.
위에 작성한 코드대로 각 메소드에 예외를 발생하는 코드만 남기고, 아래처럼만 작성해도 괜찮지 않을까 싶네요.

```suggestion
    public static void validatePlayerNumbers(String playerNumbers) {
        validateLengthAndRangeNumber(playerNumbers);
        validateNotDuplicate(playerNumbers):
    }
```